### PR TITLE
Fix Team player Bug

### PR DESCRIPTION
**Pull request by Stellation**
Hoeen's code plus the original effects. should fix the glitch that the boost doesnt show when used

### CODE REVIEW

File: mods/essb/moves.js [hunk 1]
```diff
@@ -875,52 +875,40 @@ exports.BattleMovedex = {
...
+			let stats = [];
+			for (let stat in target.boosts) {
+				if (target.boosts[stat] < 6) {
+					stats.push(stat);
+				}
+			}
+			if (stats.length) {
+				let randomStat = stats[this.random(stats.length)];
+				let boost = {};
+				boost[randomStat] = 6;
```
Comment (HoeenCoder) [inline mods/essb/moves.js]: I thought this boosted to 3 not max (6)?

Comment (Stellation) [inline mods/essb/moves.js]: ik, in exiled ssb, its boosted to max

Comment (Stellation) [inline mods/essb/moves.js]: this was in the original:

 boost[randomStat] = 6;

Comment (HoeenCoder) [inline mods/essb/moves.js]: Oh this is essb, mb

Comment (Stellation) [inline mods/essb/moves.js]: in ssbffa, its nerfed to deal with balancing as most pokes have normal movesets

**Pull request merged**